ci/test-sriov-secondary-network-aws.sh

```diff
@@ -0,0 +1,574 @@
+EOF
+}
+
+# Function to get the join command for the worker node
+function get_join_command() {
+    local control_plane_node_ip=$1
+    join_command=$(ssh -o StrictHostKeyChecking=no -i "$AWS_EC2_SSH_KEY_NAME" ubuntu@"$control_plane_node_ip" "sudo kubeadm token create --print-join-command")
+    echo "$join_command"
+}
+
+# Function to join the worker node to the cluster
+function join_worker() {
+    local worker_ip=$1
+    local join_command=$2
+    echo "Joining worker node with IP $worker_ip to the Kubernetes cluster..."
+
+    ssh -o StrictHostKeyChecking=no -i "$AWS_EC2_SSH_KEY_NAME" ubuntu@$worker_ip << EOF
+        sudo $join_command
+EOF
+}
+
+# Function to verify the Kubernetes cluster status
+function verify_cluster() {
+    local control_plane_node_ip=$1
+    echo "Verifying Kubernetes cluster status on control-plane node $control_plane_node_ip..."
+
+    ssh -o StrictHostKeyChecking=no -i "$AWS_EC2_SSH_KEY_NAME" ubuntu@$control_plane_node_ip << EOF
+        kubectl get nodes
+EOF
+}
+
+function setup_cluster {
+    # Launch Master and Worker EC2 instances
+    echo "Launching EC2 instance ..."
+    CONTROLPLANE_INSTANCE_ID=$(launch_ec2_instance "ControlPlane")
+    echo "ControlPlane EC2 instance launched with Instance ID: $CONTROLPLANE_INSTANCE_ID"
+    WORKER_INSTANCE_ID=$(launch_ec2_instance "Worker")
+    echo "Worker EC2 instance launched with Instance ID: $WORKER_INSTANCE_ID"
+
+    # Wait for EC2 instances to be fully running
+    echo "Waiting for EC2 instances to be running..."
+    aws ec2 wait instance-running --instance-ids $CONTROLPLANE_INSTANCE_ID $WORKER_INSTANCE_ID
+
+    attach_network_interface "$CONTROLPLANE_INSTANCE_ID" "control-plane"
+    attach_network_interface "$WORKER_INSTANCE_ID" "worker"
+
+    echo "======== CONTROLPLANE_INSTANCE_ID: $CONTROLPLANE_INSTANCE_ID, WORKER_INSTANCE_ID: $WORKER_INSTANCE_ID ========="
+
+    # Get the public IP addresses of the instances
+    CONTROLPLANE_IP=$(get_instance_ip "$CONTROLPLANE_INSTANCE_ID")
+    echo "Get PublicIpAddress of EC2 Instance $CONTROLPLANE_INSTANCE_ID: $CONTROLPLANE_IP"
+    WORKER_IP=$(get_instance_ip "$WORKER_INSTANCE_ID")
+    echo "Get PublicIpAddress of EC2 Instance $WORKER_INSTANCE_ID: $WORKER_IP"
+
+    echo "====== CONTROLPLANE_IP: $CONTROLPLANE_IP, WORKER_IP: $WORKER_IP ======"
+
+    # Install Kubernetes on both control-plane and worker nodes
+    install_kubernetes "$CONTROLPLANE_IP"
+    install_kubernetes "$WORKER_IP"
+
+    # Initialize Kubernetes on control-plane Node
+    initialize_control_plane_node "$CONTROLPLANE_IP"
+
+    # Get the join command and join the worker node to the cluster
+    JOIN_COMMAND=$(get_join_command "$CONTROLPLANE_IP")
+    join_worker "$WORKER_IP" "$JOIN_COMMAND"
+
+    # Verify the Kubernetes cluster status
+    verify_cluster "$CONTROLPLANE_IP"
+
+    echo "Kubernetes cluster setup completed!"
+}
+
+function build_image() {
+    chmod -R g-w build/images/ovs
+    chmod -R g-w build/images/base
```

Comment:
@wenqiq I'd recommend removing this, even if we have these commands in some other Jenkins scripts